src/DotNet.Testcontainers/Containers/ResourceReaper.cs

```diff
@@ -80,10 +96,11 @@ private ResourceReaper(Guid sessionId, string ryukImage)
     /// <summary>
     /// Starts and returns the default <see cref="ResourceReaper" /> instance.
     /// </summary>
+    /// <param name="dockerEndpoint">The Docker API endpoint.</param>
     /// <param name="ct">The cancellation token to cancel the <see cref="ResourceReaper" /> initialization.</param>
     /// <returns>Task that completes when the <see cref="ResourceReaper" /> has been started.</returns>
     [PublicAPI]
-    public static async Task<ResourceReaper> GetAndStartDefaultAsync(CancellationToken ct = default)
+    public static async Task<ResourceReaper> GetAndStartDefaultAsync(string dockerEndpoint = null, CancellationToken ct = default)
```

Comment:
`Uri endpoint = null, CancellationToken ct = default`

Reply:
Hm, I see. In the configuration, it's stored as Uri. We could extend the builder to support Uris too.